fairlearn/datasets/_fetch_acsincome.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+import pathlib
+
+from sklearn.datasets import fetch_openml
+from ._constants import _DOWNLOAD_DIRECTORY_NAME
+
+
+def fetch_acsincome(*, cache=True, data_home=None,
+                    as_frame=False, return_X_y=False):
+    """Load the ACS Income dataset.
+
+    Download it if necessary.
+
+    ==============   ==============
+    Samples total           1664500
+    Dimensionality               12
+    Features                   real
+    Target                     real
+    ==============   ==============
+
+    Source: Paper: Frances Ding (2021) [1]_
```

Comment:
Perhaps it's best to point to the Folktables repository? In any case, let's write "Ding et al." to indicate that there are multiple authors.